includes/orders/functions/orders.php

```diff
@@ -1036,7 +1039,7 @@ function edd_build_order( $order_data = array() ) {
 			// Add the adjustment.
 			$adjustment_id = edd_add_order_adjustment( $args );
 
-			edd_add_order_adjustment_meta( $adjustment_id, 'fee_id', $key );
+			edd_add_order_adjustment_meta( $adjustment_id, 'fee_id', $fee_id );
```

Comment:
Given that this is now stored in the main table, we should be able to remove this and all other such references (there are quite a few). For example:

`includes/orders/classes/class-order.php`, line 451:

```php
if ( 'fee' === $adjustment->type ) {
	$fee_id = edd_get_order_adjustment_meta( $adjustment->id, 'fee_id', true );

	$fees[ $fee_id ] = $adjustment;
}
```

We should be able to remove all instances of getting/setting this and use the new `type_key` instead.